FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -0,0 +1,478 @@
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fbc-pas-fpas;FinancialServiceProvider">
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fbc-fi-ip;PricingModel">
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fbc-fct-mkt;Exchange">
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fbc-fi-ip;CompositeMarket">
+							</rdf:Description>
+						</owl:unionOf>
+					</owl:Class>
+				</owl:onClass>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasObservedDateTime"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">security price</rdfs:label>
+		<skos:definition xml:lang="en">monetary price that some party is willing to pay, has recently paid, or would like to be paid for a given financial instrument at some point in time</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;VolumeWeightedAveragePrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;PriceAnalytic"/>
+		<rdfs:label xml:lang="en">volume-weighted average price</rdfs:label>
+		<skos:definition xml:lang="en">price determined by multiplying each trade by its volume then dividing by the volume traded for the day</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">VWAP</fibo-fnd-utl-av:abbreviation>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;VolumeWeightedOpenPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;PriceAnalytic"/>
+		<rdfs:label xml:lang="en">volume-weighted open price</rdfs:label>
+		<skos:definition xml:lang="en">price determined by multiplying each trade by its volume then dividing by the volume over a certain period during the open</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">VWOP</fibo-fnd-utl-av:abbreviation>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;Yield">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;Percentage"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;isBasedOn"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-aeq;Income"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">yield</rdfs:label>
+		<skos:definition xml:lang="en">return on the investor&apos;s capital investment</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A Yield must be based on a price, and must be in reference to some event or duration of time. It has a calculation method, and may have other qualifying terms such as for compounded yield.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
I've added most of them, but the references to the calculation method are in other, provisional ontologies and I would rather not add those now.  We could do that later if we think we have enough knowledge to do so, but it may need an SME.